cmd/src/search_stream_test.go

```diff
@@ -0,0 +1,198 @@
+package main
```

Comment:
A different way to test this would be to start a streaming server and return your mocked events, then you can just diff against the total output of src-cli search -stream. That should cover a lot more code paths and be a test that is less likely to change as you change internal details.

Reply:
Sure. Let's remove autogold then. Doing both, spinning up a stream server and using autogold is an overkill.